src/sentry/templates/sentry/emails/request-team-access.html

```diff
@@ -7,4 +7,8 @@ <h3>Request for Access</h3>
     <p><strong>{{ name }}</strong> is requesting access to the #{{ team.slug }} team on Sentry.</p>
 
     <p>Accept or decline this request in the <a href="{{ url }}">pending requests panel</a>.</p>
+
+    <p class="via">
+        You are receiving this email due because you're listed as an administrator of the #{{ team.slug }} team.
```

Comment:
wonder if `administrator` will cause confusion with the `admin` role? maybe we should just be explicit and say owner/manager? your call though

Reply:
@macqueen so technically this also sends to team admins (not just manager/owner) which is why i was more generic. We could make it specific to them, but that was more work